src/main/resources/assets/js/components/DeploymentComponent.jsx

```diff
@@ -23,8 +23,10 @@ define([
       this.setState({loading: bool});
     },
 
-    handleDestroyDeployment: function () {
-      this.props.destroyDeployment(this.props.model, this);
+    handleDestroyDeployment: function(forceStop) {
+      return function() {
+        this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);
+      }.bind(this);
```

Comment:
But I don't understand why this shouldn't work?
```
handleDestroyDeployment: function(forceStop) {
  this.props.destroyDeployment(this.props.model, {forceStop: forceStop}, this);
},
```
You shouldn't need the `return function() {}.bind(this)`-part